nnunetv2/training/loss/compound_losses.py

```diff
@@ -55,6 +59,57 @@ def forward(self, net_output: torch.Tensor, target: torch.Tensor):
 
         result = self.weight_ce * ce_loss + self.weight_dice * dc_loss
         return result
+    
+class DC_and_CE_loss_noDDP(nn.Module):
```

Comment:
How do these noDDP losses differ? The loss functions shouldn't depend on DDP or not.

Reply:
The way current losses are written, they have the DDP logic explicitly written within them with conditionals, ie. if DDP, it computes loss a certain way. We don't need that now because Lightning will handle